WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -788,19 +798,39 @@ private extension OrderListViewController {
 
     /// Sets the `topBannerView` property to an IPP feedback banner.
     ///
-    func setIPPFeedbackTopBanner() {
-        topBannerView = createIPPFeedbackTopBanner()
+    func setIPPFeedbackTopBanner(survey: SurveyViewController.Source) {
+        guard let survey = IPPsurveyVariation else {
+            return
+        }
+        topBannerView = createIPPFeedbackTopBanner(survey: survey)
         showTopBannerView()
     }
 
-    private func createIPPFeedbackTopBanner() -> TopBannerView {
+    private func createIPPFeedbackTopBanner(survey: SurveyViewController.Source) -> TopBannerView {
         let shareIPPFeedbackAction = TopBannerViewModel.ActionButton(title: Localization.shareFeedbackButton, action: { _ in
-            self.displayIPPFeedbackBannerSurvey()
+            self.displayIPPFeedbackBannerSurvey(survey: survey)
         })
 
+        var bannerTitle = ""
+        var bannerText = ""
+
+        switch survey {
+        case .IPP_COD :
+            bannerTitle = Localization.feedbackBannerTitle
+            bannerText = Localization.feedbackBannerContent
+        case .IPP_firstTransaction:
+            bannerTitle = Localization.feedbackBannerTitle2
+            bannerTitle = Localization.feedbackBannerContent2
+        case .IPP_powerUsers:
+            bannerTitle = Localization.feedbackBannerTitle3
+            bannerTitle = Localization.feedbackBannerContent3
```

Comment:
These Localisation strings would be better if they were named for the specific banners they applied to, rather than numbers. 

Alternatively, you could consider adding them to the enum somehow, but that may not be appropriate, I've not checked.

Reply:
Oops, this slipped through, changed here: d6a45ed4ea63d01e8953d6c1a8fd684eaced4a4f